_includes/manuals/1.20/1.2_release_notes.md

```diff
@@ -0,0 +1,25 @@
+## Release notes for {{page.version}}
+
+This section will be updated prior to the next release.
+
+### Headline features
+
+### Features
+
+### Bug Fixes
+
+### Deprecations
+
+* The `snippets` template macro has been deprecated and will be removed in 1.22, please use `snippet` instead.
+
+### Upgrade warnings
+
+* A migration checking for duplicate puppet classes was added. If you see `Please make sure there are no
```

Comment:
Should these be removed from nightly now?

Reply:
good point.